x-pack/test_serverless/api_integration/test_suites/common/alerting/alert_documents.ts

```diff
@@ -58,13 +65,30 @@ export default function ({ getService }: FtrProviderContext) {
     const ALERT_INDEX = '.alerts-stack.alerts-default';
     let ruleId: string;
 
+    before(async () => {
+      roleAuthc = await svlUserManager.createApiKeyForRole('admin');
```

Comment:
would it be clearer to name this `roleAdmin`?

Reply:
Just wondering if it would be clearer to name the variable `roleAdmin` instead of `roleAuthc` so we know what type of role we're passing around. There are some API tests we have (maybe not in serverless yet) where we'll create a role with non-admin privileges and use it to ensure that the APIs are respecting RBAC.

Reply:
ahh ok.  So, for instance, if I've determined the lowest possible privileged role is developer, name the token `roleDeveloper`?

Reply:
Yep exactly!

Reply:
https://github.com/elastic/kibana/pull/183381/commits/a0c440b9548ddf95a307dabc89bbce536d35de74